services/users-api/lambdas/getAttachment.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
```

Comment:
remove comment, no console.log declared in the file.

Reply:
fixed.